addons/mail/static/tests/owl/component/chat_window_manager_tests.js

```diff
@@ -601,6 +601,462 @@ QUnit.test('chat window: open / close', async function (assert) {
+                        id: i + 10,
+                        message_type: 'comment',
+                        model: 'mail.channel',
+                        record_name: 'General',
+                        res_id: i + 1,
+                    }
+                });
+            }
+            return this._super(...arguments);
+        },
+    });
+
+    // openThread
+    document
+        .querySelector(`.o_MessagingMenu_toggler`)
+        .click();
+    await testUtils.nextTick(); // re-render
+    document
+        .querySelector(`
+            .o_MessagingMenu_dropdownMenu
+            .o_ThreadPreviewList_preview`)
+        .click();
+    await testUtils.nextTick(); // re-render
+
+    // SETUP INITIAL STATE
+    // - Set a scroll position to chat window
+    document.querySelector(`.o_MessageList.o_Thread_messageList`).scrollTop = 0;
+
+    // - Set html content of the composer of the chat window
+    let composerTextInput = document.querySelector(`
+        .o_ComposerTextInput
+        > .note-editor
+        > .note-editing-area
+        > .note-editable`);
+    composerTextInput.focus();
+    document.execCommand('insertHTML', false, '<p>XDU for the win !</p>');
+
+    // - Set attachments of the composer
+    const composerLocalId = Object.keys(self.store.state.composers)[0];
+    const composerAttachmentLocalIds = [
+        this.store.dispatch('createAttachment', {
+            filename: 'blah.png',
+            id: 1,
+            name: 'blah',
+        }),
+        this.store.dispatch('createAttachment', {
+            filename: 'blah.pdf',
+            id: 2,
+            name: 'bluh',
+        }),
+    ];
+    this.store.dispatch('linkAttachmentToComposer', composerLocalId, composerAttachmentLocalIds[0]);
+    this.store.dispatch('linkAttachmentToComposer', composerLocalId, composerAttachmentLocalIds[1]);
+    await testUtils.nextTick();
+
+    // TESTS
+
+    // - phase 1 : on hiding menu
+    // -- Hide home menu
+    await this.widget.call('chat_window', 'test:will_hide_home_menu');
+    await this.widget.call('chat_window', 'test:hide_home_menu');
+
+    // -- Close chat window
+    document
+        .querySelector(`
+            .o_ChatWindow
+            .o_ChatWindowHeader
+            .o_ChatWindowHeader_commandClose`)
+        .click();
+    await testUtils.nextTick(); // re-render
+
+    // -- Reopen chat window
+    document
+        .querySelector(`.o_MessagingMenu_toggler`)
+        .click()
+    ;
+    await testUtils.nextTick(); // re-render
+    document
+        .querySelector(`
+            .o_MessagingMenu_dropdownMenu
+            .o_ThreadPreviewList_preview`)
+        .click();
+    await testUtils.nextTick(); // re-render
+
+    const scrollTopBeforeScrollToLastMessage = document.querySelector(`.o_MessageList.o_Thread_messageList`).scrollTop;
+    // "scroll" to last message
+    document.querySelector(`.o_MessageList .o_Message:last-child`).scrollIntoView();
```

Comment:
No, chat window thread already auto-scrolls to last message already.
Doesn't make sense to scroll to last message in the test.

The assertion should check that it has scrolled to the last message.
Something like the test 'auto-scroll to bottom of thread' in owl/discuss_tests.js:
https://github.com/odoo-dev/odoo/blob/master-mail-aku/addons/mail/static/tests/owl/component/discuss_tests.js#L2025

Reply:
The idea I was trying is to check that scroll to the end manually has no effect as it was done by the component. I'll check to do it in a better way :)